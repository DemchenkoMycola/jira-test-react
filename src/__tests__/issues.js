import React from 'react';
// import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import { createStore } from 'redux';
import IssuesForm from '../components/issues/AddIssue';

describe('issues', () => {

    const store = createStore(() => ({}));

    test('should render project from', () => {
        const form = renderer.create(<IssuesForm store={store} />)
        expect(form).toMatchSnapshot();
    });

});