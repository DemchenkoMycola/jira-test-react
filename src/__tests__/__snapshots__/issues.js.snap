// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`issues should render project from 1`] = `
<div
  className="issues_form"
>
  <legend>
    Add issue
  </legend>
  <label>
    Title
  </label>
  <input
    className="input"
    onChange={[Function]}
    placeholder="title"
    type="text"
  />
  <label>
    Description
  </label>
  <input
    className="input"
    onChange={[Function]}
    placeholder="description"
    type="text"
  />
  <button
    disabled={true}
    onClick={[Function]}
  >
    add
  </button>
</div>
`;
